### PR TITLE
New event statuses

### PR DESCRIPTION
**Pull request by lamrowena**
Resolves issue #37 

- Add new values to the eventName property in the EventListener object
- Added new property in the pingReturn object with the same new values

### CODE REVIEW

File: Core/CMP API Specification.md [hunk 1]
```diff
@@ -211,6 +213,21 @@ cmpId : Number, // IAB assigned CMP ID, may be 0 during stub/loading
      </td>
      </td>
   </tr>
+  <tr>
+    <td><code>'ready'</code></td>
```
Comment (patmmccann) [inline Core/CMP API Specification.md:217]: Switch this line with line 227

Comment (lamrowena) [reply Core/CMP API Specification.md]: Fixed

**Review by patmmccann — APPROVED** [review]

File: Core/CMP API Specification.md (hunk 1 above)
Comment (patmmccann) [reply Core/CMP API Specification.md]: perfect!

**Review by janwinkler — APPROVED** [review]
:-)

**Pull request merged**